src/plugins/irc-events/message.js

```diff
@@ -95,5 +96,27 @@ module.exports = function(irc, network) {
 		if ([Msg.Type.MESSAGE, Msg.Type.ACTION].indexOf(data.type) !== -1) {
 			LinkPrefetch(client, chan, msg);
 		}
+
+		// TODO: For now, do not send notifications if any client is attached
```

Comment:
Yeah, good thing to start with. We probably want to eventually add this as an option, maybe? Or do some cleverness with away, etc. But for a first version this makes perfect sense, we'll have less complaints about this feature than the other way around ;-)